src/storage/hummock_trace/src/record.rs

```diff
@@ -163,7 +164,7 @@ pub enum Operation {
     DropLocalStorage,
 
     /// Init of a local storage
-    LocalStorageInit(u64),
+    LocalStorageInit(EpochPair),
```

Comment:
The convention in this trace crate is to have a `TracedEpochPair` and impl From `EpochPair` for it, so that we don't have to add a bincode dependency in the `common` crate.